src/panels/config/automation/action/types/ha-automation-action-choose.ts

```diff
@@ -31,51 +31,58 @@ export class HaChooseAction extends LitElement implements ActionElement {
 
     return html`
       ${(action.choose ? ensureArray(action.choose) : []).map(
-        (option, idx) => html`<ha-card>
-          <ha-icon-button
-            .idx=${idx}
-            .disabled=${this.disabled}
-            @click=${this._removeOption}
-            .label=${this.hass.localize(
-              "ui.panel.config.automation.editor.actions.type.choose.remove_option"
-            )}
-            .path=${mdiDelete}
-          ></ha-icon-button>
-          <div class="card-content">
-            <h2>
+        (option, idx) => html`<ha-card outlined>
+          <ha-expansion-panel leftChevron>
+            <h3 slot="header">
               ${this.hass.localize(
                 "ui.panel.config.automation.editor.actions.type.choose.option",
                 "number",
                 idx + 1
```

Comment:
It would help if we could use a description of the conditions or actions here, as you now have to search for what option has what...

Reply:
Maybe:
```ts
${ensureArray(option.sequence)
                .map((seq) => describeAction(this.hass, seq))
                .join(", ") ||
              this.hass.localize(
                "ui.panel.config.automation.editor.actions.type.choose.option",
                "number",
                idx + 1
              )}
```